test/test_transformers.py

```diff
@@ -10,13 +10,22 @@
 import unittest
 from unittest.mock import patch, MagicMock, ANY
 import math
-from torch.backends.cuda import sdp_kernel, SDPBackend
+# from torch.backends.cuda import sdp_kernel, SDPBackend
```

Comment:
`sdp_kernel` import is commented here, but it is referenced below multiple times?